src/test/java/org/gitlab4j/api/TestProjectApi.java

```diff
@@ -672,11 +672,12 @@ public void testVariables() throws GitLabApiException {
 
         String key = TEST_VARIABLE_KEY_PREFIX + HelperUtils.getRandomInt() + "_" +  HelperUtils.getRandomInt();
         String value = "TEST_VARIABLE_VALUE_" + HelperUtils.getRandomInt() + "_" +  HelperUtils.getRandomInt();
-        Variable variable = gitLabApi.getProjectApi().createVariable(testProject, key, value, null, null);
+        Variable variable = gitLabApi.getProjectApi().createVariable(testProject, key, value, null, null, null);
 
         assertNotNull(variable);
         assertEquals(key, variable.getKey());
         assertEquals(value, variable.getValue());
+        assertFalse(variable.getMasked());
```

Comment:
@PhilippeVienne 
Not certain on this but I believe ```variable.getMasked()``` will return ```null``` in this case.

Reply:
Yes correct and that's why a second commit remove it